gc/base/standard/Scavenger.cpp

```diff
@@ -382,7 +382,11 @@ MM_Scavenger::masterSetupForGC(MM_EnvironmentStandard *env)
 	clearCycleGCStats(env);
```

Comment:
In this file, every usage of the CLI is replaced by the corresponding call to the delegate. The code should be functionally equivalent.